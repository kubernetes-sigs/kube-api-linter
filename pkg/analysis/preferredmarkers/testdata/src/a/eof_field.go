package a

type EOFFieldTest struct {
	// Field at end of file without trailing newline
	// +kubebuilder:validation:Optional
	EndOfFileField string `json:"endOfFileField"` // want `field EndOfFileField uses marker "kubebuilder:validation:Optional", should use preferred marker "k8s:optional" instead`
}