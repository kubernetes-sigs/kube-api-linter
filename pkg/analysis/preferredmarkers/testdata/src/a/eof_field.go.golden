package a

type EOFFieldTest struct {
	// Field at end of file without trailing newline
	// +k8s:optional
	EndOfFileField string `json:"endOfFileField"` // want `field EndOfFileField uses marker "kubebuilder:validation:Optional", should use preferred marker "k8s:optional" instead`
}